aws-crt-kotlin/common/src/aws/sdk/kotlin/crt/http/HttpManagerMetrics.kt

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package aws.sdk.kotlin.crt.http
+
+public data class HttpManagerMetrics(
```

Comment:
**question**: Are these connection manager metrics? If so `HttpManager` seems ambiguous and should probably be `HttpConnectionManagerMetrics` or `HttpConnectionMetrics`

**fix**: Missing docs on these metrics. I know they are meant to match SRA but probably should document them still and any differences/quirks that may lie herein.

Reply:
Similar to other Java→Kotlin mapping classes, I chose to keep this [the same as the underlying class](https://github.com/awslabs/aws-crt-java/blob/3eea28d5f617f408655054efd33278ce84d0563b/src/main/java/software/amazon/awssdk/crt/http/HttpManagerMetrics.java#L3). If you feel strongly I can change it to be clearer (which I would generally prefer when not mapping existing classes).